SmartDeviceLinkTests/DevAPISpecs/SDLLifecycleManagerSpec.m

```diff
@@ -285,7 +292,9 @@ + (void)configure:(Configuration *)configuration {
                     response.resultCode = SDLResultSuccess;
                     testManager.registerResponse = response;
                     
-                    [testManager.lifecycleStateMachine setToState:SDLLifecycleStateSettingUpHMI fromOldState:nil callEnterTransition:YES];
+                    dispatch_sync(testManager.lifecycleQueue, ^{
```

Comment:
Perhaps something similar to option 1 down here would be useful as well. Or option 2 above.